SMG.md

```diff
@@ -0,0 +1,25 @@
+# System Maintenance Guide
+
+## Quickstart - adding a new location
+1. The data for the location needs to be available via epiforecasts/covidregionaldata - check the [system maintenance guide](https://github.com/epiforecasts/covidregionaldata/blob/master/inst/smg/SMG.md) for more information
+2. Add your location to the list in R/region-list.R (in alphabetical order), setting `stable=FALSE` until testing is complete
+   ```
+   Region$new(name = "middle-earth", stable=FALSE),
+   ```
+3. Run it! `Rstudio R/run-region-updates.R -w -u -i middle-earth/*` (executing in very verbose mode, including unstable locations, only include the new location and all sub-locations)
```

Comment:
Good to give some insight here into the compute requirements/run time to expect.

Reply:
that's very dependent upon the data source?

Reply:
yes agree but there are some approximate statements that can be made. Something along the lines of subnational estimates across 20 subnational areas is likely to take approximately x time when run using y cores. Just to give users an idea if its something they can run.

Reply:
I might need you to suggest what time is needed - I'm only able to run it on non-representative hardware (a big vm in the cloud).

Reply:
16 regions takes approximately 20 minutes on a 64 core cluster. This should scale roughly linearly and so take approximately 40 minutes on a 32 core cluster, 1 hour and 20 minutes on a 16 core cluster and approximately 5 hours and 20 minutes on a 4 core computer.